packages/pds/src/db/index.ts

```diff
@@ -78,6 +122,11 @@ export class Database {
   }
 
   async close(): Promise<void> {
+    this.notifyClient?.removeAllListeners()
+    this.notifyClient?.release()
+    if (this.facets.dialect === 'pg') {
+      await this.facets.pool.end()
+    }
```

Comment:
I have feeling `db.destroy()` might do this for you (though assuming that `db` shares the same pool).

Reply:
hah i had that same feeling but got hanging tests 🙃

Reply:
Oh, huh!  Well either way, I don't think it can do any harm :)